### ISSUE TITLE
Reason is undefined, soft bounce is always 1

### ISSUE THREAD

**Issue by beeing** [sisimai/p5-Sisimai#312]
```
Final-recipient: RFC822; email@address.com
Action: failed
Status: 5.0.0
Remote-MTA: dns; 183.81.162.32
X-Supplementary-Info: <183.81.162.32 #5.0.0 smtp; 550 "Unknown User">
```
Should the above be hard bounce?

```
Final-recipient: RFC822; email@address.com
Action: failed
Status: 5.0.0
Remote-MTA: dns; subdomain.mail.protection.outlook.com
X-Supplementary-Info: <subdomain.mail.protection.outlook.com #5.0.0 smtp;
 550 5.4.1 [email@address.com]: Recipient address rejected: Access
 denied [xxx.eop-APC01.prod.protection.outlook.com]>
```
Access denied - maybe due to spam filter?

```
Final-recipient: RFC822; email@yahoo.com
Action: failed
Status: 5.0.0
Remote-MTA: dns; mta5.am0.yahoodns.net
X-Supplementary-Info: <mta5.am0.yahoodns.net #5.0.0 smtp; 554 delivery error:
 dd Sorry, your message to email@yahoo.com cannot be delivered. This
 mailbox is disabled (554.30). - mta4490.mail.gq1.yahoo.com>
```
supposed to be hard bounce?

**Comment by azumakuniyuki** [comment]
Considering only `message/delivery-status` part above, 

# The 1st is
- reason: "userunknown"
- softbounce: 0 ( it should be a "hard bounce")

# The 2nd is
- reason: "norelaying" (it may not be rejected by a spam filter, we think)
- deliverystatus: "5.4.1" https://github.com/sisimai/p5-Sisimai/blob/master/lib/Sisimai/Bite/Email/Office365.pm#L28
- softbounce: 1

# The 3rd is
- reason: "suspend" (it is not hard bounce)
- softbounce: 0

We have no enough sample emails including `X-Supplementary-Info` field. If possible, could you sent  the entire emails to our development team ? 

Best regards.

**Comment by beeing** [comment]
Have emailed to your team. Thanks!

**Comment by azumakuniyuki** [comment]
@beeing Thank you for sending sample emails, we've received 2 samples :-)

**Comment by azumakuniyuki** [comment]
@beeing we have implemented `This mailbox is disabled` into `Sisimai::Reason::Suspend` and merged #314 into `master`.

The email beginning from `0014` was correctly parsed without #314. The email beginning from `020e` was correctly parsed with updates implemented at #314.

And then, we have found another problem when we parsed emails you sent. The problem is about the value of `Subject` header in the original message and described at issue #315.

Thanks :-)
